src/utils/numberFormat.js

```diff
@@ -0,0 +1,3 @@
+export function formatNumberWithCommas(number) {
+  return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
```

Comment:
P1:
자바스크립트 Number 객체의 내장 메소드중 toLocaleString이라는 함수를 사용하시면 동일 동작합니다.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#using_tolocalestring

```suggestion
export function formatNumberWithCommas(number) {
  return number.toLocaleString()
}
```